### ISSUE TITLE
Schema SupplierOrderById: in version 3.0.0

### ISSUE THREAD

**Issue by larsolofsson** [papinet/papiNet-API#176]
**SellerProduct:**
required: - paper to be changed to - required: - paperAndBoard

countryCode is optional in classifications:
enum: - ExportHarmonisedSystemCode
Add a comment: countryCode is required for enum_-ExportHarmonisedSystemCode

gradeCode:
   description: The product identifier for a not-fully-defined product.
   change it to: The product identifier assigned by the seller for a not-fully-defined product.

partNumber:
  Add description: The product identifier assigned by the seller for a fully-defined product.

**customerArticleReference:**
gradeCode:
   description: The product identifier for a not-fully-defined product.
  change it to: The product identifier assigned by the customer for a not-fully-defined product.

partNumber:
  Add description: The product identifier assigned by the customer for a fully-defined product.

**lineItems:/quantities:**
 uom:
 The following comments should be deleted
ReelPackage # Issue #137: Renamed from "Package" (2024-08-15)
Pallet      # Issue #137: Renamed from "PalletUnit" (2024-08-15)

**Update other schema items accordingly if same issues exist.**

**Comment by patricekrakow** [comment]
1. Already done within commit [e601f69](https://github.com/papinet/papiNet-API/commit/e601f690ae5e033fc1f67e0f0bc7577e07a85962) triggered by issue #175.

2. I have updated the `description` of `classifications[].countryCode` as follow: (4 occurrences)

   ```yaml
   description: The (two-letter) value - e.g. "BE", "SE", ... - MUST exist in the ISO 3166-1 alpha-2 standard list. The `countryCode` is required when the `type` is `ExportHarmonisedSystemCode`.
   ```

3. `gradeCode.description` has been updated in both `sellerPorduct` and `customerArticleReference`. (7 occurrences)

4. `partNumber.description` has been addedin both `sellerPorduct` and `customerArticleReference`. (7 occurrences)

5. The comments have been removed. (8 occurrences)
